source/clear-linux/tutorials/kata.rst

```diff
@@ -0,0 +1,81 @@
+  .. _kata:
+
+Install Kata Containers
+#######################
+
+This tutorial describes how to install, configure and run Kata Containers on
+|CLOSIA|. Kata Containers is an open source project and community working to
+build a standard implementation of lightweight Virtual Machines (VMs) that feel
+and perform like containers, but provide the workload isolation and security
+advantages of VMs. Kata Containers is available in Clear Linux versions above 22860.
+
+Prerequisites
+*************
+
+This tutorial assumes you have installed |CL| on your host system.
+For detailed instructions on installing |CL| on a bare metal system, visit
+the :ref:`bare metal installation tutorial<bare-metal-install>`.
+
+Before you install any new packages, update |CL| with the following command:
+
+.. code-block:: bash
+
+   sudo swupd update
+
+Install Kata Containers
+***********************
+
+Kata Containers is included in the :file:`containers-virt` bundle. To install the
+framework, enter:
+
+.. code-block:: bash
+
+   sudo swupd bundle-add containers-virt
+
+Configure Docker to use Kata Containers by default
+
+.. code-block:: bash
+
+   sudo mkdir -p /etc/systemd/system/docker.service.d/
+   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
+   [Service]
+   ExecStart=
+   ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
+   EOF
+   
+Restart the Docker\* and Kata Containers systemd services
+
+.. code-block:: bash
+
+   sudo systemctl daemon-reload
+   sudo systemctl restart docker
```

Comment:
@joselamego  Is the restart required?  Doesn't the daemon-reload reload all of the configs by basically doing a stop and then a start for each service?

Reply:
I believe you are correct. I will further test, although the restart may do no harm.

Reply:
I asked for feedback on this and general agreement is that the restart is highly recommended, due to occasionally seen unexpected behavior in docker if not performed.